package.json

```diff
@@ -12,7 +12,7 @@
     "test": "karma start"
   },
   "devDependencies": {
-    "@folio/stripes-core": "thefrontside/stripes-core#master",
+    "@folio/stripes-core": "thefrontside/stripes-core",
```

Comment:
Shouldn't we point to the official repo here?

Reply:
Let's leave it on master of our fork until the next upstream release.